### PR TITLE
[Merged by Bors] - Add support to show helm release status

### PR DESCRIPTION
**Pull request by sbernauer**

### CODE REVIEW

File: src/operator.rs [hunk 1]
```diff
@@ -223,13 +225,14 @@ fn list_installed_operators(output_type: &OutputType) {
...
+            println!("OPERATOR              VERSION         NAMESPACE                      STATUS           LAST UPDATED");
             for (operator, operator_entry) in output.iter() {
                 println!(
-                    "{:21} {:15} {:24} {}",
+                    "{:21} {:15} {:30} {:16} {}",
                     operator,
                     operator_entry.version,
                     operator_entry.namespace,
+                    operator_entry.status,
                     operator_entry.last_updated
```
Comment (maltesander) [inline src/operator.rs:236]: There are crates that help with formatting tables. That would make it easier to maintain. Maybe you can have a look:
- https://github.com/devashishdxt/cli-table
- https://github.com/nukesor/comfy-table

**Review by maltesander — APPROVED** [review]
Just one proposition. LGTM otherwise.

File: src/operator.rs (hunk 1 above)
Comment (sbernauer) [reply src/operator.rs:236]: I actually have tried both of them before. I personally prefer the same output as docker and kubectl which i didn't get to work using them. For me they wasted too much space. But I'm absolutely happy to switch to any table helpers (that's the reason i already looked into them)

**Pull request closed without merging**